common/src/main/scala/com/gu/media/youtube/YouTubeClaims.scala

```diff
@@ -110,7 +111,7 @@ class YouTubeClaims(override val config: Config) extends YouTubeAccess with Logg
     }
   }
 
-  def createOrUpdateClaim(atomId: String, videoId: String, userName: String, blockAds: Boolean): Claim = {
+  def createOrUpdateClaim(atomId: String, videoId: String, blockAds: Boolean): Claim = {
```

Comment:
I wanted to make `atomId` a `MediaAtom` rather than a `String` but that meant moving the models which is probably too big for this PR...